.github/workflows/ci.yml

```diff
@@ -51,8 +53,10 @@ jobs:
       run: |
         sudo apt-get update
         sudo apt-get install kcov
+    - name: Install coverage.py
+      run: "pip install --upgrade pip && pip install coverage && pip install pytest"
     - name: Generate coverage
-      run: bin/crkcov --kcov-args --exclude-pattern=/usr/include,/usr/lib,lib/,spec/
+      run: bin/crkcov --kcov-args --exclude-pattern=/usr/include,/usr/lib,lib/,spec/ --coverage-dir ${{ github.workspace }}/coverage
```

Comment:
crkcov was cd'ing into the bin directory for `coveragepy` (`~/.local/bin`). We didn't quickly figure out why it was doing that, but this fixed the immediate symptom.